plugins/modules/cloud/opennebula/one_template.py

```diff
@@ -0,0 +1,263 @@
+#!/usr/bin/python
+#
+# Copyright: (c) 2021, Georg Gadinger <nilsding@nilsding.org>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+# Make coding more python3-ish
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: one_template
+
+short_description: Manages OpenNebula templates
+
+requirements:
+  - pyone
+
+description:
+  - "Manages OpenNebula templates"
```

Comment:
```suggestion
  - "Manages OpenNebula templates."
```
All descriptions (except `short_description`) should consist out of proper sentences.